### ISSUE TITLE
make_avro_record_dataset official doc is malformatted

### ISSUE THREAD

**Issue by burgerkingeater** [tensorflow/io#1012]
https://www.tensorflow.org/io/api_docs/python/tfio/experimental/columnar/make_avro_record_dataset is messy
but not sure why make_avro_dataset(https://www.tensorflow.org/io/api_docs/python/tfio/experimental/columnar/make_avro_dataset) looks well formatted while their docstring seemingly follow the same pattern.
make_avro_record_dataset: https://github.com/tensorflow/io/blob/4643b2af7fb22d508eba10b4e6e7144c362eedaa/tensorflow_io/core/python/experimental/make_avro_record_dataset.py#L28

make_avro_dataset:
https://github.com/tensorflow/io/blob/0ca6966224e715b639f143598aa8a9bb70aa2b02/tensorflow_io/core/python/experimental/avro_dataset_ops.py#L542

@yongtang any idea why?

**Comment by yongtang** [comment]
@burgerkingeater I think the difference is the empty lines. the doc render needs empty lines before `Args`, `Return` and some other places to properly find out the marker.

**Comment by yongtang** [comment]
You can compare with docstring in `make_avro_dataset` and see the empty lines.

**Comment by burgerkingeater** [comment]
thanks

**Comment by burgerkingeater** [comment]
@yongtang  https://www.tensorflow.org/io/api_docs/python/tfio/experimental/columnar/make_avro_record_dataset is still malformed, is the document up to date?

**Comment by yongtang** [comment]
@MarkDaoust @lamberta @tensorflow/docs-team 

Since a new version of tensorflow-io v0.14.0 has been released, wondering if it is possible to update the documentation page on  https://www.tensorflow.org/io/api_docs as well? This will address the messy format in make_avro_record_dataset docstring issue (the issue has been resolved in the python source code though it is not appeared in web site yet).

On a separate note, since the v0.14.0 has been released, we also made a small change in audio tutorial (https://github.com/tensorflow/io/pull/1030) to switch from `pip install tensorflow-io-nightly` => `pip install tensorflow-io`.

Maybe the page https://www.tensorflow.org/io/tutorials/audio can also be updated as well?
